### DISCUSSION TITLE
Questions about customtkinter combobox fonts

### DISCUSSION THREAD

**Discussion by Gasiro** [TomSchimansky/CustomTkinter#2493] [Q&A] [answered]
Hi all,
I was trying to set the fonts for the combobox options but have no idea how to do it. And I did get some errors from the command line like "fonts_combobox not set" or "not found". Anyone have any ideas about this?
<img width="558" alt="Screenshot 2024-07-03 at 17 02 43" src="https://github.com/TomSchimansky/CustomTkinter/assets/117976338/116ee04b-cbfd-47c3-bd07-b53aa6347557">

**Comment by dipeshSam** [comment, marked as answer]
Have you tried `dropdown_font` option provided in `CTkComboBox` class? Here is a working example:

``` Python
from tkinter import Tk
from customtkinter import CTkComboBox


if __name__ == "__main__":
	app = Tk()
	
	cb = CTkComboBox(app, width=300, height=100, dropdown_font=("Arial", 30))
	cb.pack(padx=50, pady=50)
	
	
	app.mainloop()

```
Hope, this may be helpful.
Regards.

**Reply by Gasiro** [reply]
It works. Thanks a lot!

**Reply by dipeshSam** [reply]
You are welcome. Continue your great Tkinter journey😊

Regards.
